wagtail/admin/templates/wagtailadmin/base.html

```diff
@@ -27,8 +34,14 @@
                                 {{ message|safe }}
                             </li>
                         {% endfor %}
-                    </ul>
-                {% endif %}
+                    {% endif %}
+                </ul>
+                <template data-w-messages-target="templates">
+                    <li class="error" data-message-status="error">{% icon name="warning" classname="messages-icon" %}<span></span></li>
+                    <li class="warning" data-message-status="warning">{% icon name="warning" classname="messages-icon" %}<span></span></li>
+                    <li class="success" data-message-status="success">{% icon name="success" classname="messages-icon" %}<span></span></li>
+                    <li class="other">{% icon name="warning" classname="messages-icon" %}<span></span></li>
```

Comment:
We may need to move this 'other' up to the first one, so that it can be the fall back if we do not find a matching message type.

Reply:
i think that's already been default to,  `success` while destructuring the details in the controller.

Reply:
should i put that to `other`??